.travis.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+language: go
+
+go:
+  - 1.8.3
+
+install: make deps
+script: make test
+
+notifications:
+  email: false
```

Comment:
Why no emails? A user can turn them off if they don't want them.

Reply:
b/c I copied this from gotrue 😆

Reply:
they are very noisy